src/bncsutil/mutil.h

```diff
@@ -51,7 +51,7 @@
 #define    ROR(x,n)    _lrotr((x),(n))
 #else
 #ifndef ROL
-#define ROL(a,b) (((a) << (b)) | ((a) >> 32 - (b)))
+#define ROL(a,b) (((a) << (b)) | ((a) >> (32 - (b))))
 #endif
 #ifndef ROR
 #define ROR(a,b) (((a) >> (b)) | ((a) << 32 - (b)))
```

Comment:
If `ROL` is updated with parentheses, shouldn't `ROR` be updated with parentheses?

Reply:
will do more of the warn removals in a later commit, this one was just generating the most warns